src/Build.UnitTests/BackEnd/BuildManager_Tests.cs

```diff
@@ -4279,5 +4287,35 @@ public void GraphBuildCircular()
             result.OverallResult.ShouldBe(BuildResultCode.Failure);
             result.CircularDependency.ShouldBeTrue();
         }
+
+        [Fact]
+        public void GraphBuildShouldBeAbleToConstructGraphButSkipBuild()
+        {
+            var graph = Helpers.CreateProjectGraph(_env, new Dictionary<int, int[]> {{1, new[] {2, 3}}});
```

Comment:
Nit: list out parameter names to make it easier to read. You did it just below this.

Reply:
Random question: Any worry about this method being passed invalid data?

Reply:
It's a helper method for tests, if it's passed invalid data some test somewhere will fail.